apps/dashboard/components/price-container/price-container.tsx

```diff
@@ -2,18 +2,20 @@ import React from "react";
 import { Box } from "@mui/joy";
 import PriceContainerCard from "./price-card-container";
 export interface WalletData {
-  id: string;
-  walletCurrency: string;
+  __typename: string;
+  accountId: string;
   balance: number;
+  id: string;
   pendingIncomingBalance: number;
+  walletCurrency: "BTC" | "USD" ; 
 }
 
+
 export interface PriceContainerProps {
-  defaultAccountId: string;
-  walletDetails: WalletData[];
+  walletDetails:ReadonlyArray<WalletData>;
 }
 
-const PriceContainer: React.FC<PriceContainerProps> = ({ walletDetails }) => {
+const PriceContainer: React.FC<PriceContainerProps> = ({ walletDetails  }) => {
```

Comment:
you should fix those formatting issues with something like a `eslint --fix`

Reply:
Yeah, I forgot. Maybe we can automate it using something like Husky.